bot.go

```diff
@@ -1065,3 +1065,30 @@ func (bot *BotAPI) SetMyCommands(commands []BotCommand) error {
 	}
 	return nil
 }
+
+// EscapeText takes an input text and escape Telegram markup symbols.
+// In this way we can send a text without being afraid of having to escape the characters manually.
+// Note that you don't have to include the formatting style in the input text, or it will be escaped too.
+//
+// parseMode is the text formatting mode (ModeMarkdown, ModeMarkdownV2 or ModeHTML)
+// text is the input string that will be escaped
+func (*BotAPI) EscapeText(parseMode string, text string) string {
```

Comment:
This doesn't need to be part of `BotAPI`

```suggestion
func EscapeText(parseMode string, text string) string {
```

Reply:
this is right